fury/tests/test_stream.py

```diff
@@ -103,10 +102,13 @@ def test_pillow():
     centers = np.array([[0, 0, 0], [-1, 0, 0], [1, 0, 0]])
     colors = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
 
-    actors = actor.sphere(centers, colors=colors, radii=0.1)
+    actors = [actor.sphere(center, color, radii=0.1) for center, color in zip(centers, colors)]
+    print(f"Number of actors created: {len(actors)}")
```

Comment:
Can you explain this?  Not sure why you replace 1 actor to multiple actors.

Reply:
The original code used a single actor to represent all the spheres. By creating multiple actors, one for each sphere, we can better control individual properties of each sphere, such as color and position. This approach allows for more flexibility, especially if we need to update or interact with each sphere independently. Additionally, creating multiple actors can improve the clarity of the code when working with individual objects in the scene. The print statement helps confirm the correct number of actors have been created, ensuring that all spheres are accounted for.

Reply:
> we can better control individual properties of each sphere, such as color and position? This approach allows for more flexibility, especially if we need to update or interact with each sphere independently.

Unfortunately, I do not agree with this.


Also, having multiple actors impact memories and performance which is why it has been done with one actor. I would recommend to put it back as it was.

Reply:
ok

Reply:
i have done it, please check!!